README.md

```diff
@@ -69,7 +69,30 @@ source <(kompose completion bash)
 source <(kompose completion zsh)
 ```
 
-## Building
+## Development and building of Kompose
+
+
+### Downloading the latest (master) binary
+
+"Nightly" binaries are generated on a per-merge basis and uploaded to [Bintray](https://bintray.com/kompose)
+
+__Linux and macOS:__
+
+```sh
+# Linux 
+curl -L https://kompose.bintray.com/kompose/latest/:kompose-linux-amd64 -o kompose
```

Comment:
the link is not working :-( It returns file with  just `The requested path was not found.`

This link should work  https://dl.bintray.com/kedgeproject/kedge/latest/kedge-linux-amd64